core/test/services/workarea/direct_upload_test.rb

```diff
@@ -46,6 +46,64 @@ def test_validation
       refute(direct_upload.valid?)
     end
 
+    def test_ensure_cors
+      original_host = Workarea.config.host
+      Workarea.config.host = 'test.host'
+      dev_request = mock(
```

Comment:
Can we use `ActionDispatch::TestRequest` since Rails provides that for this kind of usecase? https://api.rubyonrails.org/v5.1/classes/ActionDispatch/TestRequest.html